pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -1006,6 +1007,38 @@ func (r *Reconciler) runReconcileSeedFlow(
 		)
 	}
 
+	kubeAPIServerService := kubeapiserverexposure.NewKubeAPIServerService(seedClient, r.GardenNamespace)
+	if wildcardCert == nil {
```

Comment:
Nit: Switch if and else case to make it consistent with usage in https://github.com/gardener/gardener/blob/a36c6731581a146920f4bcd31a14528fef5a2a71/pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go#L703.